lib/asset.js

```diff
@@ -27,7 +27,8 @@ class Asset {
     }
 
     get type() {
-        return this.params ? this.params.type : undefined;
+        // to stay backwards compatible, map `mimeType` or `mimetype` to `type`
+        return this.params ? (this.params.type || this.params.mimeType || this.params.mimetype) : undefined;
```

Comment:
this way after the `Asset` class instance is formed, we will have mimetype mapped to one place

Reply:
where do we use `mimeType` instead of `mimetype`. Can we normalize it there instead ? The only reference I could find is in pipeline https://github.com/adobe/asset-compute-pipeline/blob/8ed9171f27c599ef43083323dc8c0e63e3b2e014/lib/rendition.js#L166
But its not used here. 

It would be nice if we can avoid the case sensitivity in code.

Reply:
+1. make a story if needed to review existing code.

Reply:
created a new ticket: https://jira.corp.adobe.com/browse/ASSETS-338

Reply:
You are correct @pheenomenon , looks like `mimeType` is only used for renditions. I will remove this from the source mimetype options

Reply:
ah, removing support for `source.mimeType` made me find where it is used: https://github.com/adobe/asset-compute-pipeline/blob/master/lib/storage/index.js#L119

I will keep it for now and we can take a look at it when we do the ticket.